client/src/update.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Post_date = styled.div`
+	color: #56707F;
+	font-size: 11px;
+	letter-spacing: 2px;
+	margin: 5px 0 0 0;
+	line-height: 14px;
+	padding-top: 2px;
+` 
+
+const Title = styled.div`
+	color: white;
+	font-size: 24px;
+	line-height: 29px;
+`
+
+const Update_col = styled.div`
+	width: 47%;
+	display: inline-block;
+`
+
+const Update_container = styled.div`
+	height: 195px;
+	overflow: hidden;
+`
+
+const Img_style = styled.img`
+	max-width: 100%;
+`
+
+const imgSwitch = (props) => {
```

Comment:
What does this control? Random display of an update? e.g. sometimes 1 update sometimes 2, sometimes none?

If so that kind of logic will ultimately want some sort of BE to be feeding it that information. Like DOTA has 239 updates you can click View All to see - it'd ultimately need an API call to get. Not sure if i'm misunderstanding though.

Reply:
I probably should've explained this in comments but this is just to randomize whether an image is displayed or not. I did it because I wanted to simulate a story that doesn't have an image and to be able to handle that case visually. Since it's dummy data they all have randomly generated images but irl some stories have images and some don't. Alternatively I could add a boolean on the obj in the db that determines whether it gets an image but for demonstration purposes I want to be able to see what my page looks like when a story has an image or doesn't

Reply:
hmm you're right, there are some posts without pictures - weird. I do wonder what they do when it's on the front page of their updates - just no pic? Default placeholder? Weird. Regardless okay makes sense 👍